### PR TITLE
Contact Form fix

### PR DESCRIPTION
**Pull request by Dolybim**
I've fixed the contact form, pls review

### CODE REVIEW

**Review by OluyideO — APPROVED** [review]

**Comment by Oluyide** [comment]
There is conflict on this branch

**Review by Oluyide — CHANGES_REQUESTED** [review]
Please resolve conflicts

File: src/pages/Contact.tsx [hunk 1]
```diff
@@ -1,297 +1,212 @@
...
+import fluid from "../img/fluid.jpeg";
+import {Card, Row, Col, Form, Button} from 'react-bootstrap';
+import axios from 'axios';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import '../contact.css';
+import contactImg from "../img/contactImg.jpg";
+import fb_logo from "../img/fb_logo.png";
```
Comment (Oluyide) [inline src/pages/Contact.tsx:12]: Let us use this same method as we did in the footer

Comment (Oluyide) [reply src/pages/Contact.tsx:12]: For all the socials

**Pull request closed without merging**